xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/layout.less

```diff
@@ -233,13 +233,20 @@
 
 #xwikieditor .main {
   background-color: @xwiki-page-content-bg;
-  box-shadow: 0 1px 2px @xwiki-border-color;
+  box-shadow: 0 1px 2px @input-border;
 }
 
 #xwikieditcontent .main {
   background-color: @xwiki-page-content-bg;
 }
 
+#xwikieditcontent #content {
+  vertical-align: bottom;
+  border-radius: 0;
+  border-color: @input-border;
+  resize: vertical;
```

Comment:
Not supported by IE, see https://caniuse.com/#feat=css-resize
Is this needed?

Reply:
Without it:
![9](https://user-images.githubusercontent.com/2213999/40920808-ed2cb1e8-680d-11e8-9cf3-baf9267e572e.gif)

Reply:
If IE doesn't have a resize at all then it's not a problem.